FEEDBACK.md

```diff
@@ -0,0 +1,9 @@
+General comments/feedback:
+
+Overall think it looks great. Clear and easy to follow, and plenty of good content. Here's some general feedback. Let me know if any of these sound good and I can make separate PRs for them:
+
+1. Should we include a brief overview of programming in general at the very beginning? High level, ‘what is a program?’ sort of stuff?
```

Comment:
one thing (it sucks, sorry) is that if we make a new section we have to update every single page's sidebar-nav b/c it's a static site and i was too lazy (or not lazy enough?) to use a static site generator to build the site.